volume/store/store.go

```diff
@@ -309,16 +309,20 @@ func (s *VolumeStore) create(name, driverName string, opts, labels map[string]st
 // This makes sure there are no races between checking for the existence of a volume and adding a reference for it
 func (s *VolumeStore) GetWithRef(name, driverName, ref string) (volume.Volume, error) {
 	name = normaliseVolumeName(name)
```

Comment:
When Lock() encounters a `cancelable` locker, it can Panic. This will ensure that the developer is conscious of choosing a cancel-able Lock vs a normal Lock. What we dont want is to turn all locks into cancelable locks.

If developer doesnt catch error, then Unlock will result in runtime error. So its okay.